starter/js/atm.js

```diff
@@ -1,3 +1,36 @@
+$(document).ready(function(){
+$('input #deposit').on("click", depositMoney)
+$('input #withdrawal').on("click", depositMoney)
```

Comment:
This selector is choosing anything with an id of #withdrawal that is inside an input. I believe you wanted the input itself so you'd want to select `'input#withdrawCheckings'`. Also, I think you may have misspelled the id here ;)